packages/rendermime/src/rendermime.ts

```diff
@@ -336,6 +363,17 @@ namespace RenderMime {
   }
 
   /**
+   * A rendered widget.
```

Comment:
We are using the word `rendered` in many places. Let's add a bit more detail about what is meant by the ready promise. When exact does it resolve? After attach? Later?